lib/country_select/tag_helper.rb

```diff
@@ -15,7 +15,8 @@ def country_option_tags
 
         if priority_countries_options.map(&:second).include?(option_tags_options[:selected])
           option_tags_options[:selected] = nil
-        end
+        end unless option_tags_options[:selected].kind_of?(Array)
```

Comment:
this is very hard to read, imo

essentially it's saying `if x` and then later `unless y`

Can you move the `unless` into the `if` block _or_ combine it with the `if` block?